lib/facter/util/partitions/linux.rb

```diff
@@ -55,6 +55,13 @@ def self.filesystem(partition)
       end
     end
     
+    def self.label(partition)
+      label = nil
```

Comment:
Actually that should ensure that if there's no label everything works as
always. I can try to see if even without this assignment that scenario is
covered as wel
On 13 Jan 2015 03:38, "Michael Smith" <notifications@github.com> wrote:

> In lib/facter/util/partitions/linux.rb
> <https://github.com/puppetlabs/facter/pull/836#discussion-diff-22840224>:
>
> > @@ -55,6 +55,13 @@ def self.filesystem(partition)
> >        end
> >      end
> >
> > +    def self.label(partition)
> > +      label = nil
>
> This looks like an orphaned variable. Can it be removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/facter/pull/836/files#r22840224>.
>